tools/bin/commands/util/openshift_funcs

```diff
@@ -475,6 +475,7 @@ rules:
   resources:
   - deploymentconfigs
   - deploymentconfigs/scale
+  - deploymentconfigs/finalizers
```

Comment:
Not needed to add it here. All permissions are applied through the operator. We rather remove this method and fix references to it

Reply:
Would rather avoid doing this big a change in a backport. So will simply revert this file for now.

```
commands/minishift:        add_user_permissions_for_operator "$user" "true"
commands/install:        local result=$(add_user_permissions_for_operator "$user_to_prepare" $(hasflag --cluster))
commands/util/openshift_funcs:add_user_permissions_for_operator() {
install-syndesis:add_user_permissions_for_operator() {
install-syndesis:    result=$(add_user_permissions_for_operator "$user_to_prepare" $(hasflag --cluster))
```